android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -611,6 +617,18 @@ public void setPrivacyPolicyUrl(final String urlString) {}
     @ReactMethod()
     public void setTermsOfServiceUrl(final String urlString) {}
 
+    @ReactMethod()
+    public void setUserSegmentField(final String name)
```

Comment:
Can we move this to right under `#pragma mark - Data Passing`?

Reply:
Done.